samcli/local/apigw/local_apigw_service.py

```diff
@@ -287,7 +287,9 @@ def _construct_event(flask_request, port, binary_types):
                                       resource=endpoint,
                                       request_context=context,
                                       query_string_params=query_string_dict,
+                                      multi_value_query_string_params=flask_request.args.to_dict(False),
```

Comment:
Can you add the key to the arg here? It will make this line easier to ready/understand.

Just thinking (typing) out load here. We currently have a function to get the Query strings (`LocalApigwService._query_string_params()`). This function is already iterating over the `request.args`. We can adjust this slight to either:
* Pass in the `flask_request.args.to_dict(flat=False)` and iterate over that.
* Keep the parameter the same (`flask_request`) and return two values (`multi_value_query_string_params` and `query_string_params`.

I am ok with either. I think this might help keep all the query string related things into one place, is my thought. 

Also, remove the comment on like 280-282. This is no longer relevant. - Github won't let me comment on those lines for whatever reason :(.

Reply:
Returning query_string_dict, multi_value_query_string_dict from _query_string_params.